packages/myst-cli/src/build/site/logger.ts

```diff
@@ -29,6 +29,7 @@ export function createServerLogger(session: ISession): LoggerDE {
       if (!line) return;
       // This is a spurious Remix warning https://github.com/remix-run/remix/issues/2677
       if (line.includes('is not listed in your package.json dependencies')) return;
+      if (line.includes('was not found in your node_modules')) return;
```

Comment:
The phrasing of this spurious warning changed in a recent release. This is still a problem when working in npm workspaces, but isn't actually real.